### PR TITLE
feat: add get_observation and get_observation_for_place in client

### PR DESCRIPTION
**Pull request by bbesset**
This new feature fetches current observation from the v2/observation API endpoint.

### CODE REVIEW

**Comment by bbesset** [comment]
hello @oncleben31 @dx44 @Quentame, could you please review ?

File: src/meteofrance_api/client.py [hunk 1]
```diff
@@ -74,6 +75,31 @@ def search_places(
...
+    #
+    # Observation
+    # 
+    def get_observation(
+        self,
+        latitude: float,
+        longitude: float,
+        language: str = "fr",
+    ) -> Observation:
+        resp = self.session.request(
```
Comment (Quentame) [inline src/meteofrance_api/client.py:88]: May you add a documentation ?

File: src/meteofrance_api/client.py [hunk 2]
```diff
@@ -74,6 +75,31 @@ def search_places(
...
+            params={"lat": latitude, "lon": longitude, "lang": language})
+        return Observation(resp.json())
+
+
+    def get_observation_for_place(
+        self,
+        place: Place,
+        language: str = "fr",
+    ) -> Observation:
+        return self.get_observation(place.latitude, place.longitude, language)
```
Comment (Quentame) [inline src/meteofrance_api/client.py:100]: May you add a documentation ?

File: src/meteofrance_api/model/observation.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+"""Weather observation Python model for the Météo-France REST API."""
+import sys
+
+if sys.version_info >= (3, 8):
+    from typing import TypedDict  # pylint: disable=no-name-in-module
+else:
+    from typing_extensions import TypedDict
+	
+class Observation:
+	def __init__(self, data: TypedDict):
```
Comment (Quentame) [inline src/meteofrance_api/model/observation.py:11]: Your model is nice, but other are not built like that.
Could you make this model as the others ?

And also add a doc.

**Review by Quentame — CHANGES_REQUESTED** [review]
Nice PR !
But need few changes 😅 

That would be awesome if you could add some tests 😉

File: src/meteofrance_api/client.py (hunk 1 above)
Comment (bbesset) [reply src/meteofrance_api/client.py:102]: You are right.
I just did.

File: src/meteofrance_api/client.py (hunk 2 above)
Comment (bbesset) [reply src/meteofrance_api/client.py:126]: You are right.
I just did.

File: src/meteofrance_api/model/observation.py (hunk 3 above)
Comment (bbesset) [reply src/meteofrance_api/model/observation.py]: I commited an update (new model + doc).
Please review whenever you can.

**Review by Quentame — APPROVED** [review]
Nice, thanks 🎉

**Pull request merged**